src/EventListener/DataContainer/NewsCategoryListener.php

```diff
@@ -43,18 +45,24 @@ class NewsCategoryListener implements FrameworkAwareInterface
      */
     private $session;
 
+    /**
+     * @var Slug
+     */
+    private $slug;
+
     /**
      * NewsCategoryListener constructor.
      *
      * @param Connection        $db
      * @param PermissionChecker $permissionChecker
      * @param SessionInterface  $session
      */
-    public function __construct(Connection $db, PermissionChecker $permissionChecker, SessionInterface $session)
+    public function __construct(Connection $db, PermissionChecker $permissionChecker, SessionInterface $session, ?Slug $slug)
```

Comment:
PHP 5.6 is still supported in composer.json, so please change this one to `Slug $slug = null`.

Reply:
Though you could increase it in the next minor version. Would make things easier ;).

Reply:
I will certainly have to do it, yes, but I would like it to come alongside with some real improvements (e.g. strict types usage or method return types).